src/lib/stellar/transactions.js

```diff
@@ -1,4 +1,5 @@
-import { TransactionBuilder, Networks, Server, Operation, Asset, Memo } from 'stellar-sdk'
+import { TransactionBuilder, Networks, Operation, Asset, Memo } from '@stellar/stellar-sdk'
+import Server from '@stellar/stellar-sdk'
```

Comment:
Again, you should just be able to change `Server` (in the original code) to `Horizon`, and then use `new Horizon.Server()` when instantiating it.